pandas/plotting/_core.py

```diff
@@ -3337,19 +3337,87 @@ def kde(self, bw_method=None, ind=None, **kwds):
 
     def area(self, x=None, y=None, **kwds):
         """
-        Area plot
+        Draw a stacked area plot.
+
+        An area plot displays quantitative data visually.
+        This function wraps the matplotlib area function.
 
         Parameters
         ----------
-        x, y : label or position, optional
-            Coordinates for each point.
-        `**kwds` : optional
+        x : label or position, optional
+            Coordinates for the X axis. By default uses the index.
+        y : label or position, optional
```

Comment:
As @dukebody says, for both `x` and `y`, in the first line we should have the type, more than a short description. It's a bit tricky, as `label` can be anything, but I think we're using something like `object or int, optional`. Then, we can sure clarify that it's the `label` or the `position` in the description.

Reply:
I kept it consistent with  `DataFrame.plot`, which is the more complete docstring example I could find.